tf_encrypted/tensor/int100.py

```diff
@@ -259,6 +262,9 @@ def lift(x) -> 'Int100Tensor':
         if isinstance(x, Int100Tensor):
             return x
 
+        if isinstance(x, Int100SeededTensor):
+            return x.expand()
```

Comment:
afaict this could get called every time the seeded tensor is used? ie expanded tensors are not cached?

Reply:
Yeah I think this needs to removed and explicitly expand in the mask plus mul functions. We can definitely make this better in the future with backing tensor caching.

Reply:
Well... maybe it can stay there for the mul function but I think if we explicitly expand inside the mask function we can cache the expanded tensors in the PondMaskedTensor.